### PR TITLE
Feature/df 519  agb management

### PR DESCRIPTION
**Pull request by emina-str**

### CODE REVIEW

File: alv-portal-ui/src/app/admin/admin-routing.module.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import { LegalTermsManagementComponent } from './legal-terms-management/legal-terms-management.component';
 
 const routes: Routes = [
   {
     path: 'user-info',
     component: UserInfoComponent,
+  },
+  {
+    path: 'legal-terms-management',
+    component: LegalTermsManagementComponent,
```
Comment (fabiangotzen) [inline alv-portal-ui/src/app/admin/admin-routing.module.ts]: Please also add the titleKey here like so: 
`     path: 'user-info',
    component: UserInfoComponent,
    data: {
      titleKey: 'portal.admin.user-info.user-info.title',
      authorities: [UserRole.ROLE_SYSADMIN, UserRole.ROLE_ADMIN]
    }
  },`

Also, the authority for the admin route is set in app-routing.module.ts, if the feature is also for the supporters we should add
`      authorities: [UserRole.ROLE_SYSADMIN, UserRole.ROLE_ADMIN]` as well.

File: alv-portal-ui/src/app/shared/layout/main-navigation/menu-entry.service.ts [hunk 2]
```diff
@@ -54,6 +54,12 @@ const MENU_ENTRIES: Array<MenuEntry> = [
     labelKey: 'portal.admin.user-info.menu-entry',
     path: ['admin', 'user-info'],
     userPredicate: (u) => hasAnyAuthorities(u, [UserRole.ROLE_ADMIN, UserRole.ROLE_SYSADMIN])
+  },
+  {
+    icon: 'balance-scale',
+    labelKey: 'portal.admin.legal-terms-management.title',
+    path: ['admin', 'legal-terms-management'],
+    userPredicate: (u) => hasAnyAuthorities(u, [UserRole.ROLE_ADMIN, UserRole.ROLE_SYSADMIN])
```
Comment (fabiangotzen) [inline alv-portal-ui/src/app/shared/layout/main-navigation/menu-entry.service.ts]: To clarify, but I think this feature is only meant for Sysadmins

Comment (emina-str) [reply alv-portal-ui/src/app/shared/layout/main-navigation/menu-entry.service.ts]: Here states both ADMIN and SYSADMIN roles.
https://github.com/alv-ch/jobroom2/blob/develop/src/main/webapp/app/layouts/navbar/navbar.component.html#L211

File: alv-portal-ui/src/app/admin/admin-routing.module.ts (hunk 1 above)
Comment (emina-str) [reply alv-portal-ui/src/app/admin/admin-routing.module.ts]: Fixed.

File: alv-portal-ui/src/app/admin/legal-terms-management/legal-terms-detail-modal/legal-terms-detail-modal.component.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <alv-date-input label="portal.admin.legal-terms-management.header.effectiveAt"
+                    alvFormControlName="effectiveAt"
+                    [readonly]="readonly"
+                    [minDate]="minEffectiveAt"
+                    [validationMessages]="[{ error: 'pastEffectiveAt', message: 'portal.admin.legal-terms-management.legal-terms-details.validate.pastEffectiveAt' }]"
+                    showGroupErrors="true"
+                    placement="bottom-right">
+    </alv-date-input>
+    <alv-input-field label="portal.admin.legal-terms-management.header.linkDe"
+                     class="reverse-ellipsis"
```
Comment (birom) [inline alv-portal-ui/src/app/admin/legal-terms-management/legal-terms-detail-modal/legal-terms-detail-modal.component.html]: I think the ``reverse-ellipsis`` does not work anymore. I would just use ``overflow: auto`` to make sure that the whole link is readable.

Comment (birom) [inline alv-portal-ui/src/app/admin/legal-terms-management/legal-terms-detail-modal/legal-terms-detail-modal.component.html]: I think the ``reverse-ellipsis`` does not work anymore. I would just use ``overflow: auto`` to make sure that the whole link is readable.

Comment (emina-str) [reply alv-portal-ui/src/app/admin/legal-terms-management/legal-terms-detail-modal/legal-terms-detail-modal.component.html]: With just `overflow: auto` it still gets over the form. Any hints how to fix it? maybe @99999989 

agb-management-overflow-view

Comment (emina-str) [reply alv-portal-ui/src/app/admin/legal-terms-management/legal-terms-detail-modal/legal-terms-detail-modal.component.html]: Fixed.
agb-management-overflow-view-changed

**Pull request merged**